datastore/google/cloud/datastore/_gax.py

```diff
@@ -71,6 +73,13 @@ def _grpc_catch_rendezvous():
     """
     try:
         yield
+    except GaxError as exc:
+        error_code = exc_to_code(exc.cause)
+        error_class = _GRPC_ERROR_MAPPING.get(error_code)
+        if error_class is None:
+            raise
+        else:
+            raise error_class(exc.cause.details())
```

Comment:
Note this violates DRY but once all 6 datastore methods are on GAPIC we will just delete the `exceptions.GrpcRendezvous` code